pom.xml

```diff
@@ -19,7 +19,7 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
         <!-- runtime/test dependencies -->
-        <quarkus.version>1.13.2.Final</quarkus.version>
+        <quarkus.version>1.13.6.Final</quarkus.version>
```

Comment:
Consider using version that acutaly exist.

Reply:
OK.